### ISSUE TITLE
Cameras with recordings are not being deleted.

### ISSUE THREAD

**Issue by ijunaidfarooq** [evercam/evercam-media#435]
Junaid says it works as expected for all cams with no history.
For cams with history they are transferred to a special account, scheduled for deletion, but never deleted.
Locally, this issue doesn't happen.

**Comment by marcoherbst** [comment]
More info needed please.

**Comment by ijunaidfarooq** [comment]
Those cameras who have no history are being deleted.
But all those who have the history(recordings) they are just assigned to evercam admin as an owner but not getting deleted. Also, there are no errors for that. I have tried to replicate it in local machine as well but recordings got deleted. And camera as well

https://github.com/evercam/evercam-media/blob/master/web/controllers/camera_controller.ex#L331-L343

the above code is being used for this purpose right now. I tried to replicated the same issue in local machine but failed to recognize any issue.
